maths-club-pack/content/facilitator/fence-around-a-field.md

```diff
@@ -0,0 +1,22 @@
+# Fence Around A Field
```

Comment:
I think you accidentally missed this one. Notes should be available
![image](https://user-images.githubusercontent.com/10515065/91468224-88480f00-e846-11ea-8c85-45637722439a.png)